zero-gravity-computer/issue-tracker-backend
# Move rest functionality into django_instant_rest

Instead of maintaining modules that deliver REST API functionality, we're relying on the [django_instant_rest](https://github.com/zero-gravity-computer/django_instant_rest) package to provide it.

Testing Steps:
* Pull the feature branch: `git fetch && git checkout use_django_instant_rest_package`
* Remove Pipenv virtual environment: `pipenv --rm`
* Re-install: `pipenv install`
* Run Server: `pipenv run python manage.py runserver`

Create, Read, Update, and Delete functionality should still work for all models.
